mlmpfr.opam

```diff
@@ -8,35 +8,25 @@ bug-reports:  "https://github.com/thvnx/mlmpfr/issues"
 license:      "LGPL-3.0"
 dev-repo: "git+https://github.com/thvnx/mlmpfr.git"
 build: [
-  ["cc" "test.c" "-lmpfr" "-o" "test_installed"]
-  ["./test_installed"]
```

Comment:
This have to stay I guess since I have to check that the installed library is the good one.

Reply:
Well, what happens if it's not the good one ? It's going to fail anyway, right ?

Reply:
Hum..., yes :D, at least it will at runtime. This test was a request from the opam-repo maintainers, so I will keep it. And the users can know why the installation is failing with this test.